src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -720,6 +723,92 @@ public void testDeleteOrderLineByIdNotCorrespondingToOrderId() {
     assertTrue(message.contains(lineId));
   }
 
+  @Test
+  public void testGetOrderLineById(TestContext ctx) {
+    logger.info("=== Test Get Orderline By Id ===");
+
+    String orderId = "d79b0bcc-DcAD-1E4E-Abb7-DbFcaD5BB3bb";
+
+    final CompositePoLine resp = RestAssured
+      .with()
+      .header(X_OKAPI_URL)
+      .header(X_OKAPI_TENANT)
+      .get(rootPath + "/" + orderId + "/lines/" + ANOTHER_PO_LINE_ID_FOR_SUCCESS_CASE)
```

Comment:
Please use `String.format(LINE_BY_ID_PATH, orderId, ANOTHER_PO_LINE_ID_FOR_SUCCESS_CASE)` so the URL path is defined in one place